tests/conftest.py

```diff
@@ -14,7 +14,7 @@ def __init__(self, id, username, password):
         self.password = password
 
     def __str__(self):
-        return "User(id='%s')" % self.id
+        return "User(id='%s')" % self.user_id
```

Comment:
I hate to be nitpicky on code style, but let's avoid [old style](https://docs.python.org/3/library/string.html#string-formatting) string formatting.

Reply:
Yeap. In fact, I just saw that _if called_, the `__str__` function would raise a `KeyError`. But the main point is: the method not even used in the test cases ... Shouldn't we simply drop it?

Reply:
Sure